src/ext/java/org/opentripplanner/ext/vehicletostopheuristics/VehicleToStopSkipEdgeStrategy.java

```diff
@@ -70,12 +72,13 @@ public boolean shouldSkipEdge(State current, Edge edge) {
         current.getVertex() instanceof TransitStopVertex stopVertex &&
         !stopsCounted.contains(stopVertex.getStop().getId())
       ) {
+        // TODO: 2022-12-05 filters: check performance on that and verify that this is right. Previously we were filtering just on modes
```

Comment:
Is this method even used on OTP2?

Reply:
I do not know.

Reply:
Yes, it is used.

Reply:
Is it not possible to convert the filters to a set of allowed modes?